L/libbensolve/build_tarballs.jl

```diff
@@ -0,0 +1,55 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libbensolve"
+version = v"2.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://users.fmi.uni-jena.de/~sa67leb/bensolve/files/bensolve-$(version).tgz", "f13c6708974553e346c5426c0471f395fc4de417b55ac9fb07978fa9d268f0a0")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/bensolve*/
+gcc -shared -fPIC -std=c99 -O3 -o libbensolve.$dlext *.c -lglpk -lm
+install_license doc/manual.pdf
+cp libbensolve.$dlext ${libdir}
+cp *.h ${includedir}
```

Comment:
```suggestion
cc -shared -fPIC -std=c99 -O3 -o libbensolve.$dlext *.c -lglpk -lm
install -Dvm 755 libbensolve.$dlext -t ${libdir}
install -Dvm 644 *.h -t ${includedir}
install_license doc/manual.pdf
```
Isn't there a better license file than a PDF version of the manual?

Reply:
License is at the end of the manual but there is no other that ships with this library.

Reply:
Use
```
install_license /usr/share/licenses/GPL-3.0+
```